frontend/src/console/components/CoursesManage/styles.scss

```
@@ -0,0 +1,80 @@
+@import "~styles/theme";
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@600&display=swap');
+
+.courses-page {
+    // Only put this because the page didn't pick up the right font family.
+    font-family: 'Montserrat';
```

No, the font is Chivo. Not sure why the mockups use montserrat, but if you look at the high fidelity mockup, the font used is `Chivo-Regular`: https://app.zeplin.io/project/5ea850e04326be273e4bcf5a/screen/5ffea8fc52670720f44aebf8
```suggestion
    font-family: $chivo-font;
```

You should make all fonts chivo on this page for now, and let them be adjusted once this page is migrated to the new page strucuture. Otherwise the fonts will not be consistent in the current experience.